docs/using-the-compiler.rst

```diff
@@ -614,8 +614,9 @@ Error Types
 10. ``Exception``: Unknown failure during compilation - this should be reported as an issue.
 11. ``CompilerError``: Invalid use of the compiler stack - this should be reported as an issue.
 12. ``FatalError``: Fatal error not processed correctly - this should be reported as an issue.
-13. ``Warning``: A warning, which didn't stop the compilation, but should be addressed if possible.
-14. ``Info``: Information that the compiler thinks the user might find useful, but is not dangerous and does not necessarily need to be addressed.
+13. ``YulException``: Error during Yul Code generation.
```

Comment:
```suggestion
13. ``YulException``: Error during Yul Code generation -  this should be reported as an issue.
```
We shouldn't see those usually right? They imply an internal error

Reply:
Yea. Added this change.